src/NimBLEL2CAPChannel.h

```diff
@@ -0,0 +1,115 @@
+//
+// (C) Dr. Michael 'Mickey' Lauer <mickey@vanille-media.de>
+//
+#pragma once
+#ifndef NIMBLEL2CAPCHANNEL_H
+#define NIMBLEL2CAPCHANNEL_H
+
+#include "inttypes.h"
+#include "host/ble_l2cap.h"
+#include "os/os_mbuf.h"
+#include "os/os_mempool.h"
+
```

Comment:
Unfortunately some nimble headers define `min` and `max` which causes errors when you include stl headers after them. So `#undef min` and `#undef max` will need to be inserted here to fix the build.